README.md

```diff
@@ -76,10 +76,16 @@ We also aim to be able to select any network type with cli flags.
 The CLI can work in regtest mode, by locally running a `zcashd` and `lightwalletd`.
 
 For example:
-Create the directories `~/.zcash/zd/` and `~/.zcash/ld`
-`zcashd` works in regtest with the following invocation, run from your `zcash` directory (explicit path, substitute your own username):
-`./src/zcashd --printtoconsole -conf=/home/username/.zcash/zd/zcash.conf --datadir=/home/username/.zcash/zd/`
-and the following in a `zcash.conf` file in the specified directory:
+Starting in your `zingolib/` directory.
+`mkdir reg`
+`cd reg`
+Then, copy your compiled `zcashd` `zcash-cli` and `lightwalletd` binaries to this directory.
+Make a stand-in .zcash directory: `mkdir .zc`
+`cd .zc`
+Create these sub-directories: `mkdir zd && mkdir ld`
+`cd zd`
+`touch zcash.conf`
```

Comment:
from my reading, you end up with:

```
reg
├── lightwalletd
├── .zc
│   ├── ld
│   └── zd
│       └── zcash.conf
├── zcash-cli
└── zcashd
```

I think it makes sense to just have this setup already but without the binaries?